### PR TITLE
Fieldset forms component

### PR DESCRIPTION
**Pull request by aaschlote**
Ticket: https://app.asana.com/0/1207022303033556/1208087966238694

### CODE REVIEW

File: app/services/cms/__test__/getStrapiEntryFromApi.test.ts [hunk 1]
```diff
@@ -25,8 +25,8 @@ describe("services/cms", () => {
     };
     const defaultResponseData = { data: { data: dataResponse } };
     const emptyResponseData = { data: [] };
-    const expectedRequestUrl = `${API_URL}pages?populate=deep&locale=de`;
-    const expectedStagingRequestUrl = `${API_URL}pages?populate=deep&locale=sg`;
+    const expectedRequestUrl = `${API_URL}pages?populate=deep,10&locale=de`;
+    const expectedStagingRequestUrl = `${API_URL}pages?populate=deep,10&locale=sg`;
```
Comment (pgurusinga) [inline app/services/cms/__test__/getStrapiEntryFromApi.test.ts:29]: I know it's still a draft :D but what is this `10` for? and would this break others?

Comment (aaschlote) [reply app/services/cms/__test__/getStrapiEntryFromApi.test.ts:29]: We use a plugin called [strapi-plugin-populate-deep](https://www.npmjs.com/package/strapi-plugin-populate-deep). The default value for max depth level is 5, but with my new component the level is 6, so I increased to 10.

We shouldn't have any issues.

**Review by rbrtrfl — COMMENTED** [review]
Since this branch is almost two months old and isn't being touched, I would suggest to delete it and just keep the branch until and if the topic becomes important again. wdyt?

**Comment by pgurusinga** [comment]
@aaschlote should we close this PR if we decide to move to Strapi 5?

**Comment by aaschlote** [comment]
> @aaschlote should we close this PR if we decide to move to Strapi 5?

We will move to Strapi 5, but I can close it and reopen again later

**Pull request closed without merging**